docs/test-specifications/crypto-service/AccountCreateTransaction.md

```diff
@@ -33,13 +33,13 @@ https://docs.hedera.com/hedera/sdks-and-apis/rest-api
 | Parameter Name            | Type                                             | Required/Optional | Description/Notes                                                                                                                           |
 |---------------------------|--------------------------------------------------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
 | key                       | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
-| initialBalance            | int64                                            | optional          | Units of tinybars                                                                                                                           |
+| initialBalance            | string                                            | optional          | Units of tinybars                                                                                                                           |
 | receiverSignatureRequired | bool                                             | optional          |                                                                                                                                             |
-| autoRenewPeriod           | int64                                            | optional          | Units of seconds                                                                                                                            |
+| autoRenewPeriod           | string                                            | optional          | Units of seconds                                                                                                                            |
 | memo                      | string                                           | optional          |                                                                                                                                             |
-| maxAutoTokenAssociations  | int32                                            | optional          |                                                                                                                                             |
+| maxAutoTokenAssociations  | string                                            | optional          |                                                                                                                                             |
```

Comment:
Do we need to change `int32` types as well?

Reply:
Not needed.